inferno-core/inferno-core.cabal

```diff
@@ -38,7 +38,6 @@ library
     , Inferno.Parse.Commented
     , Inferno.Parse.Error
     , Inferno.Instances.Arbitrary
-    , Inferno.Utils.QQ.Script
```

Comment:
Removed this module as we don't use it anywhere. An appendix from the old frontend perhaps?